lib/ibandit/iban.rb

```diff
@@ -52,6 +59,15 @@ def bban
       iban[4..-1] unless iban.nil?
     end
 
+    def pseudo_iban
```

Comment:
Discussed - we'll leave it as is for now.  Will make the callsite slightly more awkward, but since pseudo-IBANs shouldn't leak out to the user, we want to make usage of them explicit.